doc/nrf/app_dev/device_guides/thingy91/thingy91_updating_fw_programmer.rst

```diff
@@ -39,7 +39,7 @@ Modem firmware
 
 The :file:`CONTENTS.txt` file in the extracted folder contains the location and names of the different firmware images.
 
-The instructions in this section show you how to program the :ref:`connectivity_bridge` and :ref:`asset_tracker_v2` applications, as well as the modem firmware.
+The instructions in this section show you how to program the :ref:`connectivity_bridge` and Asset Tracker v2 applications, as well as the modem firmware.
```

Comment:
Can we link to the add-on here or remove this reference altogether? Or if we have to keep it, mention where to find the app?

Reply:
This is changed to nRF cloud multi-service sample in the whole guide.